### PR TITLE
Tickets/sitcom 758 - Prepare for Vent

### PR DESCRIPTION
**Pull request by isotuela**
HI Tiago and Yijung, 
I'd appreciate your time and feedback for this page on Prepare for Vent. The last figure is not updated, but will do tomorrow as soon as I can run it on TTS. 
I wanted to have your preliminary input and ideas. 
Many thanks! 
Ioana

### CODE REVIEW

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+   If wind speed is above or close to 8 m/s, keep vent gates and extraction fan off. 
+
+   .. figure:: ./_static/PrepareforVent_AuxTel_VentGate3andFan.png
+      :name: Dome Vent Gate 3 and Extraction Fan 
+
+      AuxTel dome vent gate #3 and extraction fan with its controller located at the dome pier. 
+
+#. Visually confirm in LOVE that the system is venting. 
+        - Telescope is closer to horizontal pointing; az = 180, el = 30 deg, rot = 0. 
+        - M1 Mirror cover is closed.
```
Comment (ykangastro) [inline Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:120]: But on the image (Daytime-Operations/Auxiliary-Telescope/AT-Venting/_static/PrepareforVent_AuxTel.png) , the M1 Mirror cover is opened

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+#. If the wind speed is below 8 m/s, manually open vent gate #3 using the remote controller and turn on the extraction fan power to max.  
+   If wind speed is above or close to 8 m/s, keep vent gates and extraction fan off. 
+
+   .. figure:: ./_static/PrepareforVent_AuxTel_VentGate3andFan.png
+      :name: Dome Vent Gate 3 and Extraction Fan 
+
+      AuxTel dome vent gate #3 and extraction fan with its controller located at the dome pier. 
+
+#. Visually confirm in LOVE that the system is venting. 
+        - Telescope is closer to horizontal pointing; az = 180, el = 30 deg, rot = 0.
```
Comment (ykangastro) [inline Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:119]: It is not mandatory, but if there is any chance of replacing the figure, "Daytime-Operations/Auxiliary-Telescope/AT-Venting/_static/PrepareforVent_AuxTel_LOVEdisplayConfirmation.png" would be better to have the same pointing position with az = 180, el = 30 deg, rot = 0 (for me, I prefer to check images and setup the same when I run any procedures).

**Review by ykangastro — CHANGES_REQUESTED** [review]
Hi Ioana, thank you for your documentation. It is really great to see and learn what is the procedure for prepare_for_vent. Most of the descriptions on the document seem fine to me. I just left some comments and/or suggestions which are not that critical. 

Please let me know if there is anything that I missed to review.

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst (hunk 2 above)
Comment (isotuela) [reply Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst]: 👍, completely agree , I mistakenly uploaded the prepare for on-sky screenshot. Solved that now.

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst (hunk 1 above)
Comment (isotuela) [reply Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:121]: Yes, sorry,  I wrongly uploaded the prepare for on-sky screenshot. Solved that now.

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+   The script performs the following steps:
+       - Check that all components are enabled. 
+       - Disable dome following. 
+       - Home the dome. 
+       - Disable AOS open loop corrections. 
+       - Close M1 primary mirror cover. 
+       - Close dome, if it was open. 
+       - Enable AOS open loop correction.
+       - Point telescope horizontally; az = 180, el = 30, rot = 0 deg.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:95]: Agree. Thanks! 👍

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+.. _Prepare-for-vent-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is venting. 
+
+  The ``preparefor/vent.py`` script will run until the Sun's elevation is 5 degrees above the horizon, if there's no operator interaction prior to that. 
+  
+  No earlier than 2.5 hours before sunset and no later than 1 hour before sunset, :ref:`prepare for on-sky operations <AT-On-sky-WEP>`.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:63]: Good idea 👍 I will also leave it in the post-condition as it's a different condition:)

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+:ref:`calibrations <LATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>`, when applicable, and no later than 2.5 hours before sunset. 
+
+.. _Prepare-for-vent-Prerequisites:
+
+Prerequisites
+=============
+
+- Before taking the decision to open, review the weather conditions and :ref:`weather constraints <Nighttime-Operations-Weather-Constraints>` page.
+
+- Assumes observatory is fully ready to operate and all components are enabled.
```
[the remark this answers is not in the document]
Comment (isotuela) [reply Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst]: 👍 Modified

**Review by ykangastro — APPROVED** [review]
For me, now all looks good if my suggestions will be applied to the pushed version. I will set it to "approve" for now just in case.

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+========
+
+This procedure gets the AuxTel system ready for venting, positioning the telescope horizontally and the dome to face opposite the Sun to maximize the venting of the system. 
+
+Venting accelerates thermal equalization, increasing the amount of free air currents. 
+This way, the local thermal imbalances in the air inside the dome, that could deteriorate image quality, are minimized.  
+
+In a typical observing night, observers must start venting in the afternoon following the execution of :ref:`daytime checkout <AT-Daytime-Checkout-Daytime-Checkout>` and 
+:ref:`calibrations <LATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure>`, when applicable, and no later than 2.5 hours before sunset. 
+If less than 2.5 hours before sunset, please proceed to :ref:`prepare for on-sky operations <AT-On-sky-WEP>`.
```
Comment (patrickingraham) [inline Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:37]: This should probably be based on the sun's elevation.

File: Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+.. _Prepare-for-vent-Post-Condition:
+
+Post-Condition
+==============
+
+- Telescope is venting. 
+
+  The ``auxtel/prepare_for/vent.py`` script will run until the Sun's elevation is 5 degrees above the horizon, if there's no operator interaction prior to that. 
+  
+  No earlier than 2.5 hours before sunset and no later than 1 hour before sunset, :ref:`prepare for on-sky operations <AT-On-sky-WEP>`.
```
Comment (patrickingraham) [inline Daytime-Operations/Auxiliary-Telescope/AT-Venting/Prepare-for-Vent.rst:63]: here too

**Comment by patrickingraham** [comment]
Thanks for making the changes. Please make sure you rebase and squash child commits.

**Review by patrickingraham — APPROVED** [review]
Made one small comment. Please be sure to rebase and squash. Let me know if you need help

**Pull request merged**